cmd/vicadmin/vicadm.go

```diff
@@ -123,6 +131,15 @@ func init() {
 	}
 
 	extraconfig.Decode(src, &vchConfig)
+	if vchConfig.HostCertificate == nil {
+		log.Infoln("--no-tls is enabled")
```

Comment:
If this line appears in the vicadmin logs, we can change this to `log.Infoln("--no-tls is enabled on the personality")` to be clearer.